html/usage-d3.html

```diff
@@ -0,0 +1,220 @@
+        
+    </div>
+    <svg width="800" height="700"></svg>
+
+    <div>&nbsp;</div>
+    <div class="mx-4">
+        <h3><strong>About this graph</strong></h3>
+        <p>
+            This graph shows a snapshot (<em>between <span id="from"></span> and <span id="to"></span></em>) of library usage (unsorted) when using Binary mode or Execution of Cpp libraries (red bars),
+            and for the Rust libraries (blue bars) it shows all usage.
+        </p>
+        <h4>Libraries not displayed on the chart</h4>
+        <p>
+            <ul id="notOnChart"></ul>
+
+        </p>
+        <p>
+            It does not contain all library usage, none of the header-only libraries are included, and some libraries are not stored in conan (openssl, hdf5, libuv, lua, nsimd).
+        </p>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/5.7.0/d3.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/underscore@1.13.4/underscore-umd-min.js"></script>
+    
+    <script language="Javascript">
+        const urlprefix = 'https://conan.compiler-explorer.com';
+        let chart;
+        let cpplibraries;
+        let rustlibraries;
+        const minRange = 20;
+
+        function loadChart() {
+            
+            d3.csv('https://compiler-explorer.s3.amazonaws.com/public/library_usage.csv')
+                .then(data => {
+                    makeChart(data)
+                });
+        }
+
+        function loadCppLibraries(done) {
+            fetch(urlprefix + '/libraries/cpp' , {
+                    method: 'GET',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    }
+                })
+                .then(response => response.json())
+                .then(info => {
+                    cpplibraries = Object.keys(info)
+                    done[0](done.slice(1));
+                })
+        }
+
+        function loadRustLibraries(done) {
+            
+            fetch(urlprefix + '/libraries/rust', {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json'
+                }
+            })
+            .then(info => {
```

Comment:
this is missing `.then(response => response.json())`

Reply:
yea fixed it in the PR #22